### PR TITLE
Add support for Custom Ballot size per Badgeholder

### PR DESCRIPTION
**Pull request by carlbarrdahl**
- Fix PAR-213

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -73,6 +73,11 @@ model Ballot {
     @@unique([voterId, roundId])
 }
 
+enum RoundType {
+    impact
+    project
+}
+
```
Comment (carlbarrdahl) [inline prisma/schema.prisma:80]: The deployed schema already had this round type but there was a difference in the checked in schema.

Comment (hussedev) [reply prisma/schema.prisma:80]: :wink:

**Review by hussedev — DISMISSED** [review]
good job!!!!

**Review by hussedev — APPROVED** [review]
LGTM

**Pull request merged**